src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -377,6 +377,11 @@ private void WatchProjectFiles(ProjectFileInfo projectFileInfo)
                 QueueProjectUpdate(projectFileInfo.FilePath, allowAutoRestore: true, projectFileInfo.ProjectIdInfo);
             });
 
+            _fileSystemWatcher.Watch(".editorconfig", (file, changeType) =>
```

Comment:
should this watch on `projectFileInfo.AnalyzerConfigFiles` instead?

Reply:
Would it still need a catch-all to handle new .editorconfigs being added to subfolders? I expected this would be easy way of handling Add and Update.